src/status_im/ui/screens/chat/utils.cljs

```diff
@@ -27,7 +27,7 @@
   (or (and (= from current-public-key)
            [react/text {:style (style true)}
             (i18n/label :t/You)])
-      (format-author from style nil)))
+      (format-author from style username)))
```

Comment:
unfortunately I tried that quick fix already and the problem is that the alias shows up in blue instead of black, as if it was a regular username

Reply:
however a simple way to solve it is just to get rid of the option to display regular usernames, since afaik there is no such thing in v1, therefore username is either an ens username (show in blue) or a random name (show in black) @rachelhamlin @hesterbruikman is that right?

Reply:
I can probably just fix it in the meantime

Reply:
Almost:) 

The design for V1 is to get rid of the so called 'display name' that you edit under profile. Instead it includes:

- Random name shown to all (contacts + non-contacts) >>> font size:12 | weight: 400 | **grey**

**or** 

- ENS name shown to all (contacts + non-contacts) >>> font size: 13 | weight: 500 | blue

**or** 

Both random name and ENS name can be overwritten on the client side with a nickname.  Meaning I can assign a nickname to physical indelible cicada as well as to yenda.stateofus.eth after adding him to my contacts. Similarly to how I would assign a name to a phonenumber in my phonebook.
- Nickname >>> font size: 13 | weight : 500 | grey


Discuss: https://discuss.status.im/t/profile-information-structure-please-review/1080
Designs: https://www.figma.com/file/TNCyHKtR3sx5EL6YznFWUa4O/Profile?node-id=1461%3A355

cc @andmironov @errorists

Reply:
Ok, so since we are getting rid of usernames, I will just remove it from the reply, as the nick name feature won't be implemented in the scope of this task, thanks!